packages/lesswrong/components/votes/SmallSideVote.tsx

```diff
@@ -55,20 +57,31 @@ const SmallSideVote = ({ document, hideKarma=false, classes, collection }: {
   const karma = voteProps.baseScore;
 
   let moveToAlignnmentUserId = ""
+  let documentTypeName = "comment";
```

Comment:
Hmm, not a huge fan of doing this with a mutable variable. Felt like it made it harder to read (as opposed to a switch or even just two ternary statements). But doesn't seem that terrible.